src/frontend/components/report/ReportCard.vue

```diff
@@ -37,7 +24,8 @@
           <v-icon size="22" class="pr-1">
             mdi-update
           </v-icon>
-          {{ report.data.totalUpdatedDocs }} {{ $t("reportHistory.updatedDocs") }}
+          {{ report.data.totalUpdatedDocs }}
+          {{ $t("reportHistory.updatedDocs") }}
```

Comment:
You may want to pass value into `$t` by using [Vue-i18n formatting](https://kazupon.github.io/vue-i18n/guide/formatting.html#formatting)